lib/Model.php

```diff
@@ -349,13 +349,12 @@ public function __construct(array $attributes=[], $guard_attributes=true, $insta
     public function &__get($name)
     {
         // check for getter
+        $name = strtolower($name);
```

Comment:
Shouldn't this be something like:

```php
$methodName = "get_" . strtolower($name) ;
if (method_exists($methodName) {
            $retValue = $this->$name();
            return $retValue;
```

...that way you don't have to worry about stuff breaking if you make it down to line `358`, right? (maybe we should add tests around non-lowercase attribute names?)

Reply:
Also, fun TODO: make the casing configurable (eg. I use camelCase in my own codebase, not snake_case).